src/openrct2-ui/interface/Widget.cpp

```diff
@@ -920,16 +920,7 @@ namespace OpenRCT2::Ui
 
     bool WidgetIsActiveTool(const WindowBase& w, WidgetIndex widgetIndex)
```

Comment:
Probably better to add an isToolActive signature for this and replace this one entirely.